mne/viz/evoked.py

```diff
@@ -1778,20 +1778,20 @@ def _set_ylims_plot_compare_evokeds(ax, any_positive, any_negative, ymin, ymax,
 
     fraction = 2 if ax.get_ylim()[0] >= 0 else 3
 
-    if truncate_yaxis is not False:
+    if truncate_yaxis:
```

Comment:
non-empty strings evaluate to True. Also see [PEP8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations):

> Don't compare boolean values to True or False using ==.
>
> Yes:   `if greeting:`
> No:    `if greeting == True:`
> Worse: `if greeting is True:`